Pcap++/header/DpdkDevice.h

```diff
@@ -498,11 +498,13 @@ namespace pcpp
 		 * its content when done using it
 		 * @param[out] rawPacketArrLength The length of MBufRawPacket pointers array will be written into
 		 * @param[in] rxQueueId The RX queue to receive packets from
+		 * @param[in] reuse Signifies that *rawPacketsArr already contains an array of MBufRawPacket objects with length equals to rawPacketArrLength. If provided
+		 * array is big enough to store the burst of packets received from NIC it will not be reallocated, otherwise it will.
 		 * @return True if packets were received and no error occurred or false if device isn't opened, if device is currently in capture mode
 		 * (using startCaptureSingleThread() or startCaptureMultiThreads() ), if rxQueueId doesn't exist on device, or if DPDK receive packets method returned
 		 * an error
 		 */
-		bool receivePackets(MBufRawPacket** rawPacketsArr, int& rawPacketArrLength, uint16_t rxQueueId);
+		bool receivePackets(MBufRawPacket** rawPacketsArr, int& rawPacketArrLength, uint16_t rxQueueId, bool reuse = false);

```

Comment:
I saw that the unit-tests pass but unfortunately they don't include the DPDK unit-tests because I didn't manage to set up DPDK environment in Travis CI.
Could you please run them manually in your environment and verify that everything passes?